dev-libs/websocketpp/websocketpp-0.8.1.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Websocket++ header-only library"
+DESCRIPTION="WebSocket++ is a header only C++ library that implements RFC6455 \
+The WebSocket Protocol. It allows integrating WebSocket client and server \
+functionality into C++ programs. It uses interchangeable network transport \
+modules including one based on raw char buffers, one based on C++ iostreams, \
+and one based on Asio (either via Boost or standalone). End users can write \
+additional transport policies to support other networking or event libraries as needed."
+HOMEPAGE="https://github.com/zaphoyd/websocketpp"
+COPYRIGHT="2014, Peter Thorson."
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/zaphoyd/websocketpp/archive/0.8.1.zip"
+CHECKSUM_SHA256="80f49469f1702ec64ef5d8e71302f6b781dca8ae06e560421cab42961c8c7ce6"
+PATCHES="websocketpp-0.8.1.patch"
```

Comment:
Could you create a patchset for this instead of a patch file? (this would save your credentials), in case you need help for that just call out :)
Also, you can replace 0.8.1 with $portVersion (here and in the download link), this makes future updates to the recipe smaller

Reply:
I was following the Haikuports example for generating the patch. Happy to generate it in a different way - can you point me to the correct documentation?

Reply:
My bad, the patch looks ok :) you probably renamed it from *.patchset to *.patch (patchset is what's being generated by haikuporter and is what we usually use then) :)